builtin/credential/radius/path_config.go

```diff
@@ -0,0 +1,251 @@
+package radius
+
+import (
+	"strings"
+
+	"github.com/fatih/structs"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `config`,
+		Fields: map[string]*framework.FieldSchema{
+			"host": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "radius host to connect",
+			},
+
+			"port": &framework.FieldSchema{
+				Type:        framework.TypeInt,
+				Default:     1812,
+				Description: "radius port (default: 1812)",
+			},
+			"secret": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "secret shared with the radius server",
+			},
+			"allow_unknown_users": &framework.FieldSchema{
```

Comment:
The purpose of that flag is to allow users that successfully authenticate to radius to receive a basic set of capabilities without every single user having an entry in the auth/radius/users/ configuration path.

So if `allow_unknown_users` is enabled, anyone able to authenticate to RADIUS will be authenticated to vault. If disabled, authentication will be rejected by vault until an entry for the user is written.